foia_hub/settings/local_settings.py

```diff
@@ -1,13 +1,19 @@
 from .base import *
 import os
+import re
 import dj_database_url
+from cfenv import AppEnv
+
+database_url = os.getenv("DATABASE_URL")
+
+env = AppEnv()
+cf_foia_db = env.get_service(name=re.compile('foia-db'))
+if cf_foia_db:
+    database_url = cf_foia_db.credentials['uri']
```

Comment:
I don't think lines 9-12 are needed -- cloud.gov will inject a DATABASE_URL env var. Using an env var directly (rather than jumping through cfenv) reduces coupling.